style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+  padding: 0;
+  background-image: url(./assets/geo.png);
+  background-repeat: no-repeat;
+}
+
+.navbar {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+}
+
+.tri-svg {
+  position: absolute;
+  left: 0;
+  top: 0;
+  height: 70px;
+  width: 70px;
+}
+
+.sq-svg {
+  position: absolute;
+  left: 32px;
+  top: 0;
+  height: 70px;
+  width: 70px;
+}
+
+.menu {
+  margin-top: 30px;
+  margin-right: 16px;
+}
+
+.hero-section {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.heading {
+  font-family: 'Playfair Display', serif;
+  color: #1a2236;
+  font-size: 50px;
+  margin-left: 16px;
+  margin-top: 104px;
+  line-height: 50px;
+  margin-bottom: 16px;
+}
+
+.para {
+  font-family: 'Poppins', sans-serif;
+  padding: 0 16px;
+}
+
+.btn {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

<img width="279" alt="image" src="https://user-images.githubusercontent.com/52934128/198187731-380b420a-fad9-4bfe-b901-788a7cb9f6b7.png">

the button on the pressed state has the `background: #FFFFFF;`, `color: #1A2236;`, `border-width: 2px` and on hover has the `background: #FFFFFF;`, `color: #1A2236;` `border-width: 1px`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color:  #FFFFFF;;
   color: #1A2236
   border-width: 2px;
}
```
**for hover state**
```
button:hover {
    background-color: #FFFFFF;;
    color: #1A2236
    border-width: 1px;
}